projects/Mallard/src/helpers/storage.ts

```diff
@@ -62,6 +63,8 @@ const userDataCache = createAsyncCache<IdentityAuthData>('user-data-cache');
 
 const iapReceiptCache = createAsyncCache<ReceiptIOS>('iap-receipt-cache');
 
+const oktaDataCache = createAsyncCache<OktaAuthData>('okta-data-cache');
```

Comment:
Dont believe this is used as Okta manages its own cache within the Native Module. However, the types require it so passing it through and likely to be removed in tidy up